site/src/components/home/config/configDialog.jsx

```diff
@@ -118,21 +147,73 @@ const ConfigInput = ({setting, updateData, updateError}) => {
             {...baseProps}
             {...textFieldProps}
             helperText={helper}
+            onChange={(e) => {
+                if (value === '' && (setting.emptyAllowed === undefined && !setting.emptyAllowed)) {
```

Comment:
Shouldn't this check for `setting.emptyAllowed !== undefined` instead of `setting.emptyAllowed === undefined`? The error should (only) be shown when `emptyAllowed` is explicitly set to `false`. If it's `undefined` then the empty value should be silently ignored, as it is now.

Reply:
Thinking about it again, I think this `(setting.emptyAllowed === undefined && !setting.emptyAllowed)` can be replaced by `setting.emptyAllowed === false`, for simplicity.

Reply:
You are correct. Updated.